lower/llpcSpirvLowerImageOp.cpp

```diff
@@ -600,6 +810,93 @@ void SpirvLowerImageOp::visitCallInst(
     }
 }
 
+// =====================================================================================================================
+// Tracks descriptor loading to handle CFG.
+llvm::Value* SpirvLowerImageOp::TrackDescriptorLoad(
+    Value*                   pOperand,                // Value to track from
+    DescriptorType           descType,                // Descriptor type
+    ShaderImageCallMetadata* pImageCallMeta,          // [out] Image call metadata
+    ConstantInt**            ppMemoryQualifier)       // [out] Memory qualifier
+{
+    Value*       pDescriptor = nullptr;
+
+    if (isa<SelectInst>(pOperand))
+    {
+        auto pSelectInst = cast<SelectInst>(pOperand);
+        Value* pTrueValue = pSelectInst->getTrueValue();
+        Value* pFalseValue = pSelectInst->getFalseValue();
+        pTrueValue  = TrackDescriptorLoad(pTrueValue,  descType, pImageCallMeta, ppMemoryQualifier);
+        pFalseValue = TrackDescriptorLoad(pFalseValue, descType, pImageCallMeta, ppMemoryQualifier);
+        pDescriptor = SelectInst::Create(pSelectInst->getCondition(), pTrueValue, pFalseValue, "", pSelectInst);
+    }
+    else if (isa<LoadInst>(pOperand))
+    {
+        ConstantInt* pDescSet = nullptr;
+        ConstantInt* pBinding = nullptr;
+        Value*       pIndex = nullptr;
+
+        ExtractBindingInfo(cast<LoadInst>(pOperand),
```

Comment:
is it possible to inline ExtractBindingInfo? it seems it is only used by this function now. and both functions are to check the descriptor info.